README.md

```diff
@@ -1,8 +1,8 @@
 # UID2 Integration Examples
 
-The [UID2 framework](https://github.com/UnifiedID2/uid2docs/tree/main) enables publishers to integrate in either of the following ways:
+The [UID2 framework](https://unifiedid.com/docs/intro) enables publishers to integrate in either of the following ways:
```

Comment:
please confirm if this is the best page to use

Reply:
Yes, it is.